common/static/common/js/discussion/views/new_post_view.js

```diff
@@ -209,6 +209,37 @@
                     success: function(response) {
                         var thread, discussionBreadcrumbsModel;
                         thread = new Thread(response.content);
+
+                        /* This is used as a signal to the Learning MFE to determine if the user
```

Comment:
Please give feedback on these comments and if they are clear. I have a lot of context in my head, and I want to make sure it's communicated here for when this logic moves/needs updating.

Reply:
They're doing their best, but it's still a bit much for my tiny brain.

You say `to catch both inline discussions and posts on the discussion tab` -- the _why_ here is still missing for me. Especially since you say we aren't doing the discussion tab yet. This is prepping for the future?

`Unfortunately, we only care about the first time a user posts and this will send a request` -- Maybe change this to: `We only care about the first time a user posts but unfortunately, this will send a request`

So once the Discussion MFE exists, we will send the request from just the Discussion MFE? Because it's handling all discussions? Or just the discussion tab stuff and we're keeping inline discussions here? I guess some context I'm missing is what the discussion MFE is and what its scope is.